lib/main_common.pm

```diff
@@ -234,6 +235,19 @@ sub is_kernel_test {
         || get_var('TRINITY'));
 }
 
+sub get_ltp_tag {
+    my $tag = get_var('LTP_RUNTEST_TAG');
+    if (!defined $tag) {
+        my $hdd = get_var('HDD_1');
+        if (defined $hdd) {
+            $tag = basename($hdd) . '.txt';
+        } else {
+            $tag = get_var('DISTRI') . '-' . get_var('VERSION') . '-' . get_var('ARCH') . '-' . get_var('BUILD') . '-' . get_var('FLAVOR') . '-' . get_var('LTP_RELEASE') . '@' . get_var('MACHINE');
```

Comment:
I think you are missing `.txt`.

I wonder if we could just use one of these branches? Otherwise we will have a code path that most people won't test when changing something.

Reply:
OK, lets use else branch. It's more complicated than `HDD_1` variable, but the more complicated form should emulate it.